src/domain/customers/utils.ts

```diff
@@ -2,29 +2,22 @@ import { CUSTOMERS } from './__generated__/CUSTOMERS';
 
 export interface CustomerData {
   id: string;
-  firstName: string;
-  lastName: string;
-  nickname: string | null;
-  email: string | false | null;
-  phone: string | false | null;
+  name: string;
+  nickname: string;
 }
 
-export const getCustomersData = (data: CUSTOMERS | undefined | any) => {
-  if (data?.profiles?.edges) {
-    return data.profiles.edges.map(profile => {
+export const getCustomersData = (data: CUSTOMERS | undefined) => {
+  return (
+    data?.profiles?.edges.reduce<CustomerData[]>((acc, profile) => {
       if (profile?.node) {
-        const profileData: CustomerData = {
-          email: profile.node.emails[0]?.email,
-          firstName: profile.node.firstName,
+        const profileData = {
+          name: `${profile.node.firstName} ${profile.node.lastName}`,
           id: profile.node.id,
-          lastName: profile.node.lastName,
-          nickname: profile.node.nickname,
-          phone: profile.node.phones[0]?.phone,
+          nickname: profile.node.nickname || '',
         };
-        return profileData;
+        return [...acc, profileData];
       }
-      return [];
-    });
-  }
-  return [];
+      return acc;
+    }, []) ?? []
```

Comment:
Is this on purpose?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator